src/main/java/edu/tamu/app/service/TemplateService.java

```diff
@@ -1,28 +0,0 @@
-package edu.tamu.app.service;
```

Comment:
Without this being used, removing it and its unit tests are acceptable. If removing it is the choice, the dependencies brought in the pom should be removed as well.

https://github.com/TAMULib/ProjectManagementService/blob/sprint2-staging/pom.xml#L69

Thymeleaf dependency is a hefty dependency and swaps out the Spring default JSP. It also, throws warnings on bootup

Leaving it in case of use in the future would also be acceptable.